atlasdb-config/src/main/java/com/palantir/atlasdb/factory/TransactionManagers.java

```diff
@@ -451,6 +449,14 @@ private static LockAndTimestampServices createRawLeaderServices(
             Supplier<RemoteLockService> lock,
             Supplier<TimestampService> time,
             String userAgent) {
+        if (leaderConfig.leaders().size() == 1) {
```

Comment:
@schlosna TimeLock uses a different config block (and as @chrisalice points out leader block requires yourself to be part of it)